src/components/dropdown-menu/DropdownMenu.js

```diff
@@ -45,27 +46,29 @@ class DropdownMenu extends React.Component<Props, State> {
     };
 
     componentDidUpdate(prevProps: Props, prevState: State) {
+        const { shouldCloseOnEventBubbling } = this.props;
         if (!prevState.isOpen && this.state.isOpen) {
             // When menu is being opened
-            document.addEventListener('click', this.handleDocumentClick, true);
-            document.addEventListener('contextmenu', this.handleDocumentClick, true);
+            document.addEventListener('click', this.handleDocumentClick, !shouldCloseOnEventBubbling);
```

Comment:
Can we invert this condition, give it a default value of `true`, and name it `useCapture` or something similar?

Reply:
change to useBubble per discussion as we need to default to false to avoid changing snapshot and cause breaking change.